jquery-src/jquery.calendars.chinese.js

```diff
@@ -273,7 +273,7 @@
             var monthDaysTable = LUNAR_MONTH_DAYS[year - LUNAR_MONTH_DAYS[0]];
             var intercalaryMonth = monthDaysTable >> 13;
 
-            return (intercalaryMonth && intercalaryMonth === monthIndex);
+            return (intercalaryMonth !== 0 && intercalaryMonth === monthIndex);
```

Comment:
I'm not happy with this function returning a `boolean`. Originally, I decided to return a `boolean` to resemble `leapYear`, but I often find myself wanting to know the intercalary month.

How do you think? After the dust settles, shall I change it to return the intercalary month number and 0 if none. This change is mostly backwards- compatible, and it'd help me avoid unnecessary recalculations in some of the methods.